include/swift/SIL/SILInstruction.h

```diff
@@ -7845,7 +7845,7 @@ class TryApplyInst final
 };
 
 // SWIFT_ENABLE_TENSORFLOW
-/// `differentiable_function` - given a function and differentiation indices and
+/// `differentiable_function` - given a function, differentiation indices and
```

Comment:
Minor: Oxford comma please (used below for `linear_function`) 🙂 
```suggestion
/// `differentiable_function` - given a function, differentiation indices, and
```